packages/mdc-menu-surface/README.md

```diff
@@ -204,25 +209,25 @@ Method Signature | Description
-`getAnchorDimensions() => {width: number, height: number, top: number, right: number, bottom: number, left: number}` | Returns an object with the dimensions and position of the anchor (same semantics as `DOMRect`).
-`getBodyDimensions() => {width: number, height: number}` | Returns an object with width and height of the body, in pixels.
-`getWindowDimensions() => {width: number, height: number}` | Returns an object with width and height of the viewport, in pixels.
-`getWindowScroll() => {x: number, y: number}` | Returns an object with the amount the body has been scrolled on the `x` and `y` axis.
-`setPosition(position: {top: string, right: string, bottom: string, left: string}) => void` | Sets the position of the menu surface element.
+`getInnerDimensions() => MenuDimensions` | Returns an object with the items container width and height.
+`getAnchorDimensions() => ClientRect \| null` | Returns an object with the dimensions and position of the anchor.
+`getBodyDimensions() => MenuDimensions` | Returns an object with width and height of the body, in pixels.
+`getWindowDimensions() => MenuDimensions` | Returns an object with width and height of the viewport, in pixels.
+`getWindowScroll() => MenuPoint` | Returns an object with the amount the body has been scrolled on the `x` and `y` axis.
+`setPosition(position: Partial<MenuDistance>) => void` | Sets the position of the menu surface element.
 `setMaxHeight(value: string) => void` | Sets `max-height` style for the menu surface element.
 
 ### `MDCMenuSurfaceFoundation`
 
 Method Signature | Description
 --- | ---
-`setAnchorCorner(corner: Corner) => void` | Sets the corner that the menu surface will be anchored to. See [constants.js](./constants.js)
-`setAnchorMargin(margin: AnchorMargin) => void` | Sets the distance from the anchor point that the menu surface should be shown.
+`setAnchorCorner(corner: Corner) => void` | Sets the corner that the menu surface will be anchored to. See [constants.ts](constants.ts)
+`setAnchorMargin(margin: Partial<MenuDistance>) => void` | Sets the distance from the anchor point that the menu surface should be shown.
 `setIsHoisted(isHoisted: boolean) => void` | Sets whether the menu surface has been hoisted to the body so that the offsets are calculated relative to the page and not the anchor.
 `setFixedPosition(isFixed: boolean) => void` | Sets whether the menu surface is using fixed positioning.
 `setAbsolutePosition(x: number, y: numnber) => void` | Sets the absolute x/y position of the menu. Should only be used when the menu is hoisted or using fixed positioning.
-`handleBodyClick(event: Event) => void` | Method used as the callback function for the `click` event.
-`handleKeydown(event: Event) => void` | Method used as the callback function for the `keydown` events.
+`handleBodyClick(event: MouseEvent) => void` | Method used as the callback function for the `click` event.
```

Comment:
There's a typo of `number` on the line before this one in the existing docs

Reply:
Done